paper.md

```diff
@@ -0,0 +1,71 @@
+---
+title: 'DSSE: An environment for simulation of reinforcement learning-empowered drone swarm maritime search and rescue missions'
+tags:
+  - Python
+  - PettingZoo
+  - reinforcement learning
+  - multi-agent
+  - drone swarms
+  - maritime search and rescue
+  - shipwrecked people
+authors:
+  - name: Renato Laffranchi Falcão
+    orcid: 0009-0001-5943-0481
+    corresponding: true
+    equal-contrib: true
+    affiliation: 1
+  - name: Jorás Custódio Campos de Oliveira
+    orcid: 0000-0000-0000-0000
+    equal-contrib: true
+    affiliation: 1
+  - name: Pedro Henrique Britto Aragão Andrade
+    orcid: 0009-0000-0056-4322
+    equal-contrib: true
+    affiliation: 1
+  - name: Ricardo Ribeiro Rodrigues
+    orcid: 0009-0008-1237-3353
+    equal-contrib: true
+    affiliation: 1
+  - name: Fabrício Jailson Barth
+    orcid: 0000-0001-6263-121X
+    equal-contrib: true
+    affiliation: 1
+  - name: José Fernando Basso Brancalion
+    orcid: 0000-0000-0000-0000
+    equal-contrib: true
+    affiliation: 2
+affiliations:
+ - name: Insper, Brazil
+   index: 1
+ - name: Embraer, Brazil
+   index: 2
+date: 19 April 2024
+bibliography: paper.bib
+
+---
+
+# Summary
+
+The goal of this project is to contribute to the research of solutions that employ reinforcement learning techniques to maritime search and rescue missions of shipwrecked people. The software equip researchers with a simulation of shipwrecked people casting away accordingly to maritime currents to produce an stochastic environment to be used to train and evaluate autonomous agents.
```

Comment:
- faltou um 's na palavra 'equip' para fazer o plural
- shipwrecked people casting away -> 'casted away'
- accordingly to maritime -> 'according' 
- apos a palavra maritime, recomendo trocar pra "producing an stochastic..."

Reply:
Correcting:

```suggestion
The goal of this project is to contribute to the research of solutions that employ reinforcement learning techniques to maritime search and rescue missions of shipwrecked people. The software equip's researchers with a simulation of shipwrecked people casted away according to maritime currents to producing a stochastic environment to be used to train and evaluate autonomous agents.
```